doc/rst/meta/compiler-internals-doxygen/queries.rst

```diff
@@ -0,0 +1,288 @@
+.. default-domain:: cpp
```

Comment:
Overall this looks awesome! Checked the docs and things look great.

One thing I don't see mentioned here is the new `mark{Unowned,Owned}Pointer`